src/chart/heatmap/HeatmapView.ts

```diff
@@ -260,10 +271,17 @@ class HeatmapView extends ChartView {
                 if (isNaN(data.get(dataDims[1], idx) as number)) {
                     continue;
                 }
+                const shape: RectLike & {
```

Comment:
Try using the type `graphic.Rect['shape']`

Reply:
copy that